airbyte-protocol/models/src/main/resources/airbyte_protocol/airbyte_message.yaml

```diff
@@ -24,6 +27,13 @@ properties:
   log:
     description: "log message: any kind of logging you want the platform to know about."
     "$ref": "#/definitions/AirbyteLogMessage"
+  spec:
+    "$ref": "../../../../../../airbyte-config/models/src/main/resources/types/ConnectorSpecification.yaml"
```

Comment:
... welp this is awful.

Reply:
could we reference this using the `$id` field of the file instead?

Reply:
This is also making me wonder why we use the absolute github paths as the ids for files instead of a "logical" name like `airbyte_config/ConnectorSpecification.yaml`

Reply:
if we try to use the url in id to do refs local development will become a nightmare.